app/views/org_admin/shared/_theme_selector.html.erb

```diff
@@ -1,5 +1,10 @@
-<%# locals: all_themes & popover_message %>
+<%# locals: all_themes, asRadio & popover_message %>
+<% asRadio = asRadio ||= false %>
```

Comment:
the memoized operator is not needed here and in any case it is used incorrectly apart from using camelCase notation which is not a convention in ruby. If you really want to use it, you should do the following:
```
asRadio ||= false
```
which does set asRadio if is falsy, i.e. nil or false.

Reply:
ayiyi, the problems of switching back and forth between JS and Ruby. ;) Will switch asRadio to as_radio